source/develop/release-management/releases/3.1/featurelist.html.md

```diff
@@ -11,8 +11,8 @@ wiki_last_updated: 2012-03-22
 | % Complete | Name                                                                               | Detailed description                                                               | Summary                                                          | Design                                      | Updated    |
 |------------|------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|------------------------------------------------------------------|---------------------------------------------|------------|
 | 0%         | [Quota](/develop/release-management/features/sla/quota/)                                               | [Quota](/develop/release-management/features/sla/detailedquota/)                                       | Quota feature for managing resource usage quota.                 | [ Quota ](Features/Design/Quota) | 2011-12-01 |
-| 0%         | Task management                                                                    | [ Task management ](Features/TaskManagerDetailed‎)                      | Improving task management.                                       |                                             | 2011-12-01 |
-| 0%         | Locking mechanism                                                                  | [ Locking mechanism ](Features/DetailedLockMechanism‎)                  | A new locking mechanism.                                         |                                             | 2011-12-01 |
+| 0%         | Task management                                                                    | [ Task management ](Features/TaskManagerDetailed)                      | Improving task management.                                       |                                             | 2011-12-01 |
```

Comment:
I tried to normalized some cases but it was not always easy so probably a lot remaining.